doc/source/index.rst

```diff
@@ -8,26 +8,70 @@ Welcome to the documentation for pyqtgraph
 
 Contents:
 
-.. toctree::
```

Comment:
Here is where I'd put a hidden toctree referring to the top-level sections. This is the cause of the warnings about those pages not being included in a toctree. Since we already have the new panels, we don't really need to show the actual toctree, so this should do it (also makes the top nav bar work):

```rst
.. toctree::
   :hidden:

   getting_started/index
   user_guide/index
   debug
   development/index
   api_reference/index
```

Not sure what the best ordering is.